src/api/cpp/cvc5.h

```diff
@@ -4184,20 +4227,22 @@ class CVC5_EXPORT Solver
                    const std::vector<Term>& initValue) const;
   /**
    * Pop (a) level(s) from the assertion stack.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( pop <numeral> )
-   * \endverbatim
+   *
+   *     ( pop <numeral> )
+   *
    * @param nscopes the number of levels to pop
    */
   void pop(uint32_t nscopes = 1) const;
 
   /**
    * Get an interpolant
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-interpol <conj> )
-   * \endverbatim
+   *
+   *     ( get-interpol <conj> )
+   *
    * Requires to enable option 'produce-interpols'.
```

Comment:
It would be awesome if we could reference the option here. In rst, it works like this:
```
:ref:`--output <lbl-option-output>`
```